packages/node/src/login/oidc/OidcLoginHandler.ts

```diff
@@ -67,24 +69,26 @@ export default class OidcLoginHandler implements ILoginHandler {
   ) {}
 
   async canHandle(options: ILoginOptions): Promise<boolean> {
-    return hasIssuer(options) && hasRedirectUrl(options);
+    return hasIssuer(options);
   }
 
-  async handle(options: ILoginOptions): Promise<void> {
+  async handle(
+    options: ILoginOptions
+  ): Promise<(ISessionInfo & { fetch: typeof fetch }) | undefined> {
     if (!hasIssuer(options)) {
       throw new ConfigurationError(
         `OidcLoginHandler requires an OIDC issuer: missing property 'oidcIssuer' in ${JSON.stringify(
           options
         )}`
       );
     }
-    if (!hasRedirectUrl(options)) {
-      throw new ConfigurationError(
-        `OidcLoginHandler requires a redirect URL: missing property 'redirectUrl' in ${JSON.stringify(
-          options
-        )}`
-      );
-    }
+    // if (!hasRedirectUrl(options)) {
```

Comment:
Is this block of commented code left in intentionally, or should it just be removed...?

Reply:
I'll clean this up as part of the housekeeping: this code should be moved elsewhere. I added a TODO note: 18b481a745bd001ffc8df6c8d436262a59968fb9